mne/preprocessing/ica.py

```diff
@@ -2084,3 +2101,230 @@ def _band_pass_filter(ica, sources, target, l_freq, h_freq, verbose=None):
         raise ValueError('Must specify both pass bands')
 
     return sources, target
+
+
+@verbose
+def corrmap(icas, template, threshold="auto", label="bads",
+            plot=True, copy=False, ch_type="eeg"):
```

Comment:
Yes, so maybe make it as in the original - if labels is not none, ICs are modified in place. Otherwise, it's a dry run. Label doubles as a verb, so it's kind of intuitive - if you tell corrmap "Label!", it labels things, otherwise it only plots.